task2.js

```diff
@@ -0,0 +1,19 @@
+const { pipeline } = require("stream");
+const fs = require("fs");
+const csv = require("csvtojson");
+
+const task2 = () => {
+  const inputFile = fs.createReadStream("./csv/nodejs-hw1-ex1.csv");
```

Comment:
Consider extracting hard-coded strings into separate constants at the top of the file.
In the production case filenames would be dynamic and paths would probably be passed down to the script from the outside (env variable or CLI argument) to make the whole thing reusable.